### ISSUE TITLE
Make X margins in mobile view larger

### ISSUE THREAD

**Issue by helins** [Convex-Dev/convex-web#150]
This might depend on phone resolution but it looks like X margins (left and right) can be increased in mobile view. `rem` unit is probably prefered, something like `0.5rem` or `1rem` max I guess.

![signal-2021-08-05-084555](https://user-images.githubusercontent.com/17478152/128304217-e25fa009-6dbf-464a-87b6-f3cfd40a7dec.jpeg)

Same comment applies for the dev site. Additionally, the scroll bar looks out of place compared to the upper picture, the marketing site, where the scroll bar is on the far right.

![signal-2021-08-05-084427](https://user-images.githubusercontent.com/17478152/128304239-6d15d44b-a9fd-43e1-8dcc-121a1f3b11e6.jpeg)

**Comment by pedrorgirardi** [comment]
@helins how does the latest version look to you?

**Comment by pedrorgirardi** [comment]
It seems code blocks cause the issue. Try the **Vision** page - it doesn't have any code block.

We might need to tweak the style of code blocks on mobile.

**Comment by helins** [comment]
Yes, looks related to code blocks. Could they have a `:max-width` of `100%` with `overflow-y: scroll` ?

I think you can decrease margins even a bit more.

But overall, I guess you should use padding instead of margins because the scroll bar is off and touches the text:

![signal-2021-08-11-123517](https://user-images.githubusercontent.com/17478152/129015258-569df887-f1de-4c24-b9dc-e7197f2c7a02.jpeg)
